src/NewsService.Data/NewsRepository.cs

```diff
@@ -1,89 +1,94 @@
-        {
-            var dbNews = _provider.News.FirstOrDefault(x => x.Id == newsId);
-
-            if (dbNews == null)
-            {
-                throw new NotFoundException("News was not found.");
-            }
-
-            request.ApplyTo(dbNews);
-            dbNews.ModifiedBy = _httpContextAccessor.HttpContext.GetUserId();
-            dbNews.ModifiedAtUtc = DateTime.UtcNow;
-            _provider.Save();
-
-            return true;
-        }
-
-        public Guid CreateNews(DbNews news)
-        {
-            _provider.News.Add(news);
-            _provider.Save();
-
-            return news.Id;
-        }
-
-        public List<DbNews> FindNews(FindNewsFilter findNewsFilter)
-        {
-            if (findNewsFilter == null)
-            {
-                throw new ArgumentNullException("search parameters not passed.");
-            }
-
-            var dbNewsList = _provider.News.AsQueryable();
-
-            if (findNewsFilter.AuthorId != null)
-            {
-                dbNewsList = dbNewsList.Where(x => x.AuthorId == findNewsFilter.AuthorId);
-            }
-
-            if (findNewsFilter.DepartmentId != null)
-            {
-                dbNewsList = dbNewsList.Where(x => x.DepartmentId == findNewsFilter.DepartmentId);
-            }
-
-            if (findNewsFilter.Pseudonym != null)
-            {
-                dbNewsList = dbNewsList.Where(x => x.Pseudonym == findNewsFilter.Pseudonym);
-            }
-
-            if (findNewsFilter.Subject != null)
-            {
-                dbNewsList = dbNewsList.Where(x => x.Subject == findNewsFilter.Subject);
-            }
-
-            return dbNewsList.ToList();
-        }
-
-        public DbNews GetNewsInfoById(Guid newsId)
-           => _provider.News.FirstOrDefault(dbNews => dbNews.Id == newsId) ??
-              throw new NotFoundException($"News with this id: '{newsId}' was not found.");
+      _provider.News.Add(news);
+      _provider.Save();
+
+      return news.Id;
     }
-}
\ No newline at end of file
+
+    public List<DbNews> FindNews(FindNewsFilter findNewsFilter)
+    {
+      if (findNewsFilter == null)
+      {
+        throw new ArgumentNullException("search parameters not passed.");
+      }
+
+      var dbNewsList = _provider.News.AsQueryable();
+
+      if (findNewsFilter.AuthorId != null)
+      {
+        dbNewsList = dbNewsList.Where(x => x.AuthorId == findNewsFilter.AuthorId);
+      }
+
+      if (findNewsFilter.DepartmentId != null)
+      {
+        dbNewsList = dbNewsList.Where(x => x.DepartmentId == findNewsFilter.DepartmentId);
+      }
+
+      if (findNewsFilter.Pseudonym != null)
+      {
+        dbNewsList = dbNewsList.Where(x => x.Pseudonym == findNewsFilter.Pseudonym);
+      }
+
+      if (findNewsFilter.Subject != null)
+      {
+        dbNewsList = dbNewsList.Where(x => x.Subject == findNewsFilter.Subject);
+      }
+
+      if (findNewsFilter.Prewiew != null)
```

Comment:
remove it, you can not get news by  this value

Reply:
done